db/curator/README.md

```diff
@@ -0,0 +1,20 @@
+# Elasticsearch management with curator
+
+These are some scripts used to do things like re-index the daily indices published by the elastic_cloud.rb worker.
+
+To run them use:
+
+```
+export CURATOR_PASSWORD=(password for the "elastic" user. ask mat)
```

Comment:
This might not be the most system-critical password, but documenting this form of supplying passwords bothers me as it's a risky practice; this will put the password into the shell's history where it could be read or retrieved later.

Historically, in some versions of Unix, any user could also read the environment variables of any process, although I'm not sure if that's a current concern.

If you have to use an environment variable in an ad-hoc context, better to do something like this:

```
IFS='' read -rs CURATOR_PASSWORD
export CURATOR_PASSWORD
```

and try to unset afterwards.

Reply:
I put it in .envrc myself.  Could tell people to do the same

Reply:
Yeah... it's tricky since there's going to be any number of ways that people want to do it depending on their usage pattern and technical literacy, etc. and the README for a couple of config scripts can't devolve into an education on Unix's security problems...

My preference would be to replace the `export` with my above example in order to help spread that particular usage pattern rather than `export`, and then mention `.envrc` if you want as well. Maybe better to write an executor script that prompts the user for a password if it can't find an environment variable set, but I think that's going to far here.

(Of course, following the same security model and logic, putting it in `.envrc` is almost equally a risk, although the file may have different permissions than the history file... All the same, I'd like to improve what I can).

Reply:
```suggestion
IFS='' read -rs CURATOR_PASSWORD && export CURATOR_PASSWORD
(type or paste in password for the "elastic" user and press return. ask mat for the password)
```